packages/react-data-grid/src/HeaderRow.js

```diff
@@ -83,12 +83,14 @@ const HeaderRow = createReactClass({
   },
 
   getSortableHeaderCell(column) {
+
     let SortableRenderer = SortableHeaderCell;
     if (column.headerRenderer !== undefined) {
       SortableRenderer = column.headerRenderer;
     }
     const sortDirection = (this.props.sortColumn === column.key) ? this.props.sortDirection : SortableHeaderCell.DEFINE_SORT.NONE;
-    const props = { columnKey: column.key, onSort: this.props.onSort, sortDirection: sortDirection };
+    const sortDescendingFirst = (column.sortDescendingFirst === undefined ) ? false : column.sortDescendingFirst;
+    const props = { columnKey: column.key, onSort: this.props.onSort, sortDirection: sortDirection, sortDescendingFirst: sortDescendingFirst };
```

Comment:
This can be cleaned up using the short hand notation
```
const props = { 
  columnKey: column.key, 
  onSort: this.props.onSort, 
  sortDirection, 
  sortDescendingFirst 
}
```